template/templates/modules/modules.j2

```diff
@@ -22,22 +23,18 @@ RUN \
         {%- else %}
   --mount=type=bind,from=stage-modules,src=/modules,dst=/tmp/modules,rw \
         {%- endif %}
-        {%- if module.module_type == "akmods" %}
-  --mount=type=bind,from=stage-akmods-{{ module.generate_akmods_info(os_version).stage_name }},src=/rpms,dst=/tmp/rpms,rw \
-        {%- endif %}
   --mount=type=bind,from=ghcr.io/blue-build/cli:{{ exports_tag }}-build-scripts,src=/scripts/,dst=/tmp/scripts/ \
-  --mount=type=cache,dst=/var/cache/rpm-ostree,id=rpm-ostree-cache-{{ recipe.name }}-{{ recipe.image_version }},sharing=locked \
-  /tmp/scripts/run_module.sh '{{ module.module_type }}' '{{ module.print_module_context() }}' \
-  && ostree container commit
+  /tmp/scripts/run_module.sh '{{ module.module_type }}' '{{ module.print_module_context() }}'
       {%- endif %}
     {%- endif %}
   {%- endfor %}
 {% endmacro %}
-{% macro stage_modules_run(modules_ext, os_version) %}
+
+
+{% macro ostree_modules_run(modules_ext, os_version) %}
```

Comment:
The original module macros here should remain unchanged. I would suggest making a new module macro for the vanilla OS.

Reply:
I split this to generic / ostree macros, because only rpm-ostree module calls require the things added by the ostree macro, VanillaOS, stages, etc., do not and probably never will require anything like `ostree container commit`. If there is some sort of integration needed like the we have with the `akmods` module, it would be trivial to split `generic` into `generic` and `vanilla` and switch the vanilla template to use the new macro.

I'm envisioning that we'd also ship a `generic` base image type, which would not add any OS-specific things to the Containerfile, and could thus be usable for basically any operating system that supports OCI images as a distribution mechanism (without extra work from us, but with extra work from the custom image maintainer).